lib/OptimizationEvolutionary/test/runtests.jl

```diff
@@ -45,4 +45,12 @@ Random.seed!(1234)
     
     #test that `store_trace=true` works now. Threw ""type Array has no field value" before.
     solve(prob, CMAES(μ = 40, λ = 100), store_trace = true)
+
+    # Test compatibility of user overload of trace! 
+    function Evolutionary.trace!(record::Dict{String, Any}, objfun, state, population, method::CMAES, options)
+        # record fittest individual
+        record["TESTVAL"] = state.fittest
+    end
+    # Make sure that both the user's trace record value, as well as `x` are stored in the trace.
```

Comment:
assigned solution to `sol` here, test was throwing error before because was using previously assigned `sol`